src/selectors.js

```diff
@@ -1,9 +1,15 @@
 import { matchPath } from "react-router"
+import isObject from 'lodash/isObject'
 
 const createSelectors = (structure) => {
-  const { getIn, toJS } = structure
-  const getLocation = state => toJS(getIn(state, ['router', 'location']))
-  const getAction = state => toJS(getIn(state, ['router', 'action']))
+  const { get, toJS } = structure
+  const getRouter = state => {
```

Comment:
We should export this selector function too and use it in https://github.com/supasate/connected-react-router/blob/master/src/ConnectedRouter.js#L29 and https://github.com/supasate/connected-react-router/blob/master/src/ConnectedRouter.js#L105-L106.